lenses/models/datamodel.py

```diff
@@ -101,16 +110,23 @@ def save(self,*args,**kwargs):
                 
     
 class Instrument(models.Model):
+
+    # language check skipped for this one until more relaxed english check
```

Comment:
The Instrument name and extended_name are hard-coded, so they have to pass code review and indeed we shouldn't check them for bad language. Same for the 'info' field, but let's just keep the check there to avoid mistakes.